theme/hot.css

```diff
@@ -1,122 +1,93 @@
-:root {
-  --hot-red-900: #b9302d;
-  --hot-red-800: #c93938;
-  --hot-red-700: #d63f3f;
-  --hot-red-600: #e84846;
-  --hot-red-500: #f75047;
-  --hot-red-400: #f15f5e;
-  --hot-red-300: #e67c7e;
-  --hot-red-200: #efa0a2;
-  --hot-red-100: #fed1d7;
-  --hot-red-50: #feecf0;
-  --hot-red: var(--hot-red-700);
-  --hot-red-rgb: 215 63 63;
 
-  --hot-darkred-900: #5b1018;
-  --hot-darkred-800: #6c2020;
-  --hot-darkred-700: #7c2e26;
-  --hot-darkred-600: #8d3a2f;
-  --hot-darkred-500: #994335;
-  --hot-darkred-400: #b15e4e;
-  --hot-darkred-300: #c87968;
-  --hot-darkred-200: #e69b8c;
-  --hot-darkred-100: #ffbead;
-  --hot-darkred-50: #ffdec8;
-  --hot-darkred: var(--hot-darkred-700);
-  --hot-darkred-rgb: 108 32 32;
+/* HOT theme for Shoelace */
 
-  --hot-light-gray: #e1e0e0;
-  --hot-light-gray-rgb: 225 224 224;
-  --hot-gray-900: #2c3038;
-  --hot-gray-800: #3f444f;
-  --hot-gray-700: #4f5765;
-  --hot-gray-600: #606a7c;
-  --hot-gray-500: #6e798d;
-  --hot-gray-400: #848c9d;
-  --hot-gray-300: #9aa1af;
-  --hot-gray-200: #b7bcc5;
-  --hot-gray-100: #d3d7dc;
-  --hot-gray-50: #eeeef0;
-  --hot-gray: var(--hot-gray-900);
-  --hot-gray-rgb: 104 112 127;
+@import '@shoelace-style/shoelace/dist/themes/light.css';
 
-  --hot-yellow-900: #ef7e1a;
-  --hot-yellow-800: #f4a628;
-  --hot-yellow-700: #f6bd30;
-  --hot-yellow-600: #f9d538;
-  --hot-yellow-500: #f7e53a;
-  --hot-yellow-400: #faea58;
-  --hot-yellow-300: #fcef77;
-  --hot-yellow-200: #fdf39d;
-  --hot-yellow-100: #fef8c4;
-  --hot-yellow-50: #fefde7;
-  --hot-yellow: var(--hot-yellow-800);
-  --hot-yellow-rgb: 250 167 30;
+@unocss preflights;
+@unocss default;
 
-  --hot-green-900: #105950;
-  --hot-green-800: #15766e;
-  --hot-green-700: #17867e;
-  --hot-green-600: #1a978f;
-  --hot-green-500: #1da49d;
-  --hot-green-400: #34b3ad;
-  --hot-green-300: #57c1bd;
-  --hot-green-200: #87d3d1;
-  --hot-green-100: #b7e4e3;
-  --hot-green-50: #e2f4f4;
-  --hot-green: var(--hot-green-300);
-  --hot-green-rgb: 69 156 160;
+:root,
+:host,
+.sl-theme-light {
+  --sl-color-primary-950: #b9302d;
```

Comment:
Wasn't this fine making the HOT CSS vars available in `hot.css`, then using them as `var(--hot-primary)` in the Shoelace css overrides?

I thought it might be useful to have the `hot.css` file available in the dist if we wish to import elsewhere that doesn't use shoelace.

If you think it's better this way, that's fine too!

Reply:
Yeah, but I don't want to define lots of global variables, we have not only colors but also other things (padding, typography, etc)

We can always replace the "--sl-" string with "--hot-" and use this CSS in a non-shoelace project.